lib/src/main/kotlin/com/swmansion/starknet/crypto/FeeUtils.kt

```diff
@@ -0,0 +1,12 @@
+package com.swmansion.starknet.crypto
+
+import kotlin.math.roundToInt
+
+object FeeUtils {
+    @JvmStatic
+    fun estimatedFeeToMaxFee(fee: Long, overhead: Double = 0.5): Long {
+        val overheadPercent = ((1 + overhead) * 100).roundToInt()
+
+        return fee * overheadPercent
+    }
```

Comment:
Shouldn't we do `((1 + overhead) * fee).roundToInt()`? Right now for 0.5 we get 150*fee - 150 times the fee, not 1.5.

Reply:
Yeah, absolutely xd fixed it